package.json

```diff
@@ -62,6 +62,7 @@
     "formik": "^2.4.6",
     "gravatar-url": "3.1.0",
     "lodash": "4.17.21",
+    "mime": "3",
```

Comment:
You couldn't make mime: 4 to work ?

Reply:
No because it's an es module and Jellyseerr uses cjs